src/main/scala/org/broadinstitute/dsde/agora/server/ServerInitializer.scala

```diff
@@ -32,8 +36,18 @@ class ServerInitializer(val config: Config) extends LazyLogging {
   }
 
   private def startWebServiceActors() {
+    implicit val timeout = Timeout(5.seconds)
```

Comment:
I am not too worried about it.  Perhaps have it be a parameter to the method with a default value.